app/models/manual.rb

```diff
@@ -0,0 +1,163 @@
+class Manual
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  attr_accessor :content_id, :base_path, :title, :summary, :body, :updated_at, :public_updated_at, :publication_state, :update_type
+
+  validates :title, presence: true
+  validates :summary, presence: true
+  validates :body, safe_html: true
+
+  def initialize(params)
+    @content_id = params.fetch(:content_id, SecureRandom.uuid)
+    @title = params.fetch(:title, nil)
+    @summary = params.fetch(:summary, nil)
+    @body = params.fetch(:body, nil)
+    @publication_state = params.fetch(:publication_state, nil)
+    @public_updated_at = params.fetch(:public_updated_at, nil)
+  end
+
+  %w{draft live redrafted}.each do |state|
+    define_method("#{state}?") do
+      publication_state == state
+    end
+  end
+
+  def updated_at
+    @updated_at ||= Time.zone.now
+  end
+
+  def updated_at=(timestamp)
+    @updated_at = Time.parse(timestamp.to_s) unless timestamp.nil?
+  end
+
+  def public_updated_at
+    @public_updated_at ||= Time.zone.now
+  end
+
+  def public_updated_at=(timestamp)
+    @public_updated_at = Time.parse(timestamp.to_s) unless timestamp.nil?
+  end
+
+  def sections=(section_content_ids)
+    @section_content_ids = section_content_ids
+  end
+
+  def sections
+    @sections ||= @section_content_ids.map { |content_id|
+      Section.find(content_id: content_id)
+    }
+  end
+
+  def organisations=(organisation_content_ids)
+    @organisation_content_ids = organisation_content_ids
+  end
+
+  OrganisationStruct = Struct.new(:content_id, :base_path, :title)
+
+  def organisations
+    @organisations ||= @organisation_content_ids.map { |content_id|
+      payload = publishing_api.get_content(content_id).to_hash
+      OrganisationStruct.new(payload["content_id"], payload["base_path"], payload["title"])
+    }
+  end
```

Comment:
Why not just call these `Organisation` and give them the same treatment as `Section` above by making them first class models?  (Even if they are tiny).